### PR TITLE
Fixed the error in the Overallstats.comprank by changing it to a Opti…

### PR DESCRIPTION
**Pull request by caemor**
…on<u32> instead of a u32. Also adapted the tests to it and changed it so that both null and some value are tested now

To solve the issue of the null value from the owapi.

### CODE REVIEW

**Comment by caemor** [comment]
Travis had a problem with a openssl command. Not sure how to fix this.

File: tests/v3/statistics.rs [hunk 1]
```diff
@@ -8,6 +8,8 @@ static TEST_STATS_REAL_QUICKPLAY: &'static str = include_str!("../../test_data/s
 static TEST_STATS_REAL_COMPETITIVE: &'static str = include_str!("../../test_data/stats_real_comp.json");
 
 
+
+
```
Comment (nabijaczleweli) [inline tests/v3/statistics.rs]: Amend/squash without these newlines and yer :+1:

**Review by nabijaczleweli — CHANGES_REQUESTED** [review]

**Comment by nabijaczleweli** [comment]
Travis breaks of its own accord occasionally, worry not.

**Review by nabijaczleweli — APPROVED** [review]

**Comment by caemor** [comment]
Is sth still missing? It seems Travis just couldn't get the the aquire of the API in time.

**Comment by nabijaczleweli** [comment]
No no, all is fine. I'm just really run up with work this week, I'll be merging later today :)

**Comment by nabijaczleweli** [comment]
Released in [`v2.0.0`](https://github.com/nabijaczleweli/OWAPI.rs/releases/tag/v2.0.0)
